spec/plans/provision.fmf

```diff
@@ -29,73 +20,8 @@ example: |
         how: virtual
         image: fedora
 
-/virtual:
-    summary: Provision a virtual machine (default)
-    description:
-        Create a new virtual machine on the localhost using
-        testcloud (libvirt). Testcloud takes care of downloading
-        an image and making necessary changes to it for optimal
-        experience (such as disabling UseDNS and GSSAPI for SSH).
-
-    example:
-      - |
-        # Provision a fedora virtual machine
-        provision:
-            how: virtual
-            image: fedora
-      - |
-        # Provision a virtual machine from a specific QCOW2 file,
-        # using specific memory and disk settings, using the fedora user,
-        # and using sudo to run scripts.
-        provision:
-            how: virtual
-            image: https://download.fedoraproject.org/pub/fedora/linux/releases/39/Cloud/x86_64/images/Fedora-Cloud-Base-39-1.5.x86_64.qcow2
-            user: fedora
-            become: true
-            # in MB
-            memory: 2048
-            # in GB
-            disk: 30
-
-    link:
-      - implemented-by: /tmt/steps/provision/testcloud.py
-
-/local:
-    summary: Use the localhost for testing
-    description:
-        Do not provision any system. Tests will be executed
-        directly on the localhost. Note that for some actions like
-        installing additional packages you need root permission or
-        enabled sudo.
-    example: |
-        provision:
-            how: local
-    link:
-      - implemented-by: /tmt/steps/provision/local.py
-      - verified-by: /tests/init/base
-
-/container:
-    summary: Provision a container
-    description:
-        Download (if necessary) and start a new container using
-        podman or docker.
-
-    example:
-      - |
-        # Use the fedora:latest image
-        provision:
-            how: container
-            image: fedora:latest
-      - |
-        # Use an image with a non-root user with sudo privileges,
-        # and run scripts with sudo.
-        provision:
-            how: container
-            image: image with non-root user with sudo privileges
-            user: tester
-            become: true
```

Comment:
This example has been lost in the migration.

Reply:
Retrieved !